src/hooks/useWalletSupportsChain/useWalletSupportsChain.ts

```diff
@@ -32,33 +32,33 @@ import {
   supportsPolygon,
   supportsThorchain,
 } from '@shapeshiftoss/hdwallet-core'
-import { isLedger } from '@shapeshiftoss/hdwallet-ledger'
 import { isMetaMask } from '@shapeshiftoss/hdwallet-shapeshift-multichain'
 import { useMemo } from 'react'
 import { useIsSnapInstalled } from 'hooks/useIsSnapInstalled/useIsSnapInstalled'
 import { selectAccountIdsByChainIdFilter } from 'state/slices/portfolioSlice/selectors'
 import { selectFeatureFlag } from 'state/slices/selectors'
 import { store, useAppSelector } from 'state/store'
 
-type WalletSupportsChainArgs = {
+type WalletDeviceSupportsChainArgs = {
   isSnapInstalled: boolean | null
-  chainAccountIds: AccountId[] // allows dynamic chain-support detection for Ledger
   chainId: ChainId
   wallet: HDWallet | null
 }
 
+type WalletSupportsChainArgs = {
+  chainAccountIds: AccountId[] // allows dynamic chain-support detection for Ledger
+} & WalletDeviceSupportsChainArgs
+
 // use outside react
-export const walletSupportsChain = ({
+export const walletDeviceSupportsChain = ({
   chainId,
-  chainAccountIds,
   wallet,
   isSnapInstalled,
-}: WalletSupportsChainArgs): boolean | null => {
+}: WalletDeviceSupportsChainArgs): boolean | null => {
   if (!wallet) return false
   // A wallet may have feature-capabilities for a chain, but not have runtime support for it
-  // e.g MM without snaps installed, or Ledger without chain account ids (meaning the user didn't connect said chain's accounts)
+  // e.g MM without snaps installed
   const hasRuntimeSupport = (() => {
-    if (Boolean(isLedger(wallet) && !chainAccountIds.length)) return false
```

Comment:
Runtime support for Ledger *is* a result of its currently connected chains and we should absolutely keep this as-is

Reply:
We'll need to clarify this when we discuss.

At runtime, the device "supports" the ability to connect several chains which are not yet connected - this is intentional. At runtime, metamask "device" support varies based on whether snaps are installed, which is a different concept to the current configuration the user has created using account management.

To try and clarify, here are the concepts I'm operating with - we can find better names if you like:

**Device supported chainIds**:
The chainIds available to account management, and to auto-fetching of accounts (for non-ledger wallets).
    a. Static device support - the device always supports these chains
    b. Runtime device support - the device has a modification installed (e.g snaps) which means the device now supports these
    
**Supported chainIds**:
The chainIds currently available to the rest of the application consisting of the chains the user has connected via account management (any by initial auto-fetching for non-ledger). This is strictly a subset of "Device supported chainIds".